git-link.el

```diff
@@ -360,11 +360,11 @@ Defaults to \"origin\"."
 
   (interactive (list (git-link--select-remote)))
   (let* ((remote-host (git-link--remote-host remote))
-	 (commit      (word-at-point))
+	 (commit      (downcase (word-at-point)))
```

Comment:
While I do program paranoid under what circumstances does git return uppercase letters in SHAs?

When `word-at-pont` returns `nil` `downcase` will signal an error.